### PR TITLE
New package: Slackadays.Clipboard version 0.9.0.1

### PR DESCRIPTION
**Pull request by bevsxyz**
Checklist for Pull Requests
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Is there a linked Issue?

Manifests
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] This PR only modifies one (1) manifest
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/doc/Authoring.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.6 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.6.0)?

Note: `<path>` is the directory's name containing the manifest you're submitting.

---

 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/184692)

### CODE REVIEW

**Comment by bevsxyz** [comment]
@microsoft-github-policy-service agree

**Comment by Slackadays** [comment]
Thanks :)

**Comment by stephengillie** [comment]
Automatic Validation ended with:

> Executable C:\Users\validator\AppData\Local\Microsoft\WinGet\Links\cb.exe returned exit code: -1073741515

Automated error analysis suggests -1073741515 may mean STATUS_DLL_NOT_FOUND - possibly due to a dependency on `Microsoft.VCRedist.2015+.x64` (or `.x86`).

(Automated response - build 897.)

**Comment by stephengillie** [comment]
Automatic Validation ended with:

> Executable C:\Users\validator\AppData\Local\Microsoft\WinGet\Links\cb.exe returned exit code: -1073741515

Automated error analysis suggests -1073741515 may mean STATUS_DLL_NOT_FOUND - possibly due to a dependency on `Microsoft.VCRedist.2015+.x64` (or `.x86`).

(Automated response - build 897.)

**Comment by stephengillie** [comment]
Automatic Validation ended with:

> Executable C:\Users\validator\AppData\Local\Microsoft\WinGet\Links\cb.exe returned exit code: -1073741515

Automated error analysis suggests -1073741515 may mean STATUS_DLL_NOT_FOUND - possibly due to a dependency on `Microsoft.VCRedist.2015+.x64` (or `.x86`).

(Automated response - build 897.)

File: manifests/s/Slackadays/Clipboard/0.9.0.1/Slackadays.Clipboard.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.6.0.schema.json
+
+PackageIdentifier: Slackadays.Clipboard
+PackageVersion: 0.9.0.1
+InstallerType: zip
+NestedInstallerType: portable
+NestedInstallerFiles:
+- RelativeFilePath: bin\cb.exe
+  PortableCommandAlias: cb
+Installers:
```
Comment (stephengillie) [inline manifests/s/Slackadays/Clipboard/0.9.0.1/Slackadays.Clipboard.installer.yaml:11]:
```suggestion
Dependencies:
  PackageDependencies:
    - PackageIdentifier: Microsoft.VCRedist.2015+.x64
Installers:
```

**Review by stephengillie — CHANGES_REQUESTED** [review]
image

File: manifests/s/Slackadays/Clipboard/0.9.0.1/Slackadays.Clipboard.installer.yaml (hunk 1 above)
Comment (bevsxyz) [reply manifests/s/Slackadays/Clipboard/0.9.0.1/Slackadays.Clipboard.installer.yaml:14]: I guess I'll need to update it separately for both the archs separately. Will push a comment soon when I am back on my laptop.

**Comment by stephengillie** [comment]
Verify:
image
I couldn't find how to have the application display the version number.

**Review by stephengillie — APPROVED** [review]

**Comment by bevsxyz** [comment]
> Verify:
> image
> I couldn't find how to have the application display the version number.

It's shown in the help section. But no dedicated version flag is there as far as I'm aware.

```
cb help
cb -h
cb --help
```

**Pull request merged**